src/material-experimental/mdc-chips/testing/chip-harness.ts

```diff
@@ -30,4 +32,29 @@ export class MatChipHarness extends ComponentHarness {
       exclude: '.mat-mdc-chip-avatar, .mat-mdc-chip-trailing-icon, .mat-icon'
     });
   }
+
+  /** Indicates if this chip can be deleted. */
+  async isRemovable(): Promise<boolean> {
+    const removeHarness = await this.locatorForOptional(MatChipRemoveHarness)();
+    return removeHarness !== null;
+  }
+
+  /** Delete a chip from the set. */
+  async remove(): Promise<void> {
+    const isRemovable = await this.isRemovable();
+    if (!isRemovable) {
+      throw new Error('This chip is not removable');
+    }
+
```

Comment:
I think that we should implement this by dispatching a "delete" `keydown` event, instead of going through the remove button since a chip can be removable without having a remove button.

Reply:
I'm trying it that way, but the delete doesn't actually seem to trigger correctly from a `'keydown'` event. Any ideas?

I can debug more easily once this gets synced into our repo.